### PR TITLE
[Rust] Improve the dependency settings

### PR DESCRIPTION
**Pull request by apepkuss**
In this PR, the following changes are introduced:

- Set the `wasmedge-macro` and `wasmedge-types` as workspace dependencies
- Support `rust-1.68` in the `bindings-rust` CI workflow

### CODE REVIEW

File: bindings/rust/wasmedge-sys/Cargo.toml [hunk 1]
```diff
@@ -19,8 +19,10 @@ parking_lot = "0.12.1"
 paste = "1.0.5"
 rand = "0.8.4"
 thiserror = "1.0.30"
-wasmedge-macro = {path = "../wasmedge-macro", version = "0.3"}
-wasmedge-types = {path = "../wasmedge-types", version = "0.4"}
+# wasmedge-macro = {path = "../wasmedge-macro", version = "0.3"}
```
Comment (hydai) [inline bindings/rust/wasmedge-sys/Cargo.toml:22]: Do we need to keep these?

File: bindings/rust/wasmedge-sdk/Cargo.toml [hunk 2]
```diff
@@ -15,9 +15,9 @@ anyhow = "1.0"
 num-derive = "0.3"
 num-traits = "0.2"
 thiserror = "1.0.30"
-wasmedge-macro = {path = "../wasmedge-macro", version = "0.3"}
+wasmedge-macro = {workspace = true}
 wasmedge-sys = {path = "../wasmedge-sys", version = "0.13", default-features = false}
```
Comment (hydai) [inline bindings/rust/wasmedge-sdk/Cargo.toml:19]: Do we need to make the sys in the workspace just like others.

**Review by hydai — CHANGES_REQUESTED** [review]

File: bindings/rust/wasmedge-sys/Cargo.toml (hunk 1 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sys/Cargo.toml:22]: Accepted. Remove the deprecated ones. Thanks!

File: bindings/rust/wasmedge-sdk/Cargo.toml (hunk 2 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sdk/Cargo.toml:19]: We don't need to do the same thing on `sys` as `macro` and `types` are used by both `sys` and `sdk`. However, `sys` is only used by `sdk`. Thanks!

**Comment by apepkuss** [comment]
@hydai The review issue has been fixed. Please check again. Thanks!

**Review by hydai — APPROVED** [review]

**Comment by apepkuss** [comment]
@hydai Thanks for the review!

**Pull request merged**